### PR TITLE
#274 Append widget parameter, implement in SystemTabsViewFactory

### PR DESCRIPTION
**Pull request by DevTchernov**

### CODE REVIEW

File: widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/core/widget/TabsWidget.kt [hunk 1]
```diff
@@ -20,6 +20,7 @@ import dev.icerock.moko.widgets.core.style.view.WidgetSize
 @WidgetDef(SystemTabsViewFactory::class)
 class TabsWidget<WS : WidgetSize>(
     private val factory: ViewFactory<TabsWidget<out WidgetSize>>,
+    val onTabSelected: ((Int) -> Unit)? = null,
```
Comment (Alex009) [inline widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/core/widget/TabsWidget.kt:23]: maybe just `val selectedTab: MutableLiveData<Int>? = null`?
and replace
```
widget.selectedTab?.value = tab?.position ?: 0
```

and add to android:
```
widget.selectedTab.bind(lifecycleOwner) { position ->
            viewPager.setCurrentItem(position)
        }
```
on ios:
```
widget.selectedTab?.bind { segmentedControl.selectedSegmentIndex = it }
```

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/core/factory/SystemTabsViewFactory.kt [hunk 2]
```diff
@@ -195,6 +196,8 @@ actual class SystemTabsViewFactory actual constructor(
             container.bottomAnchor.constraintEqualToAnchor(bottomAnchor).active = true
         }
 
+
+
```
Comment (Alex009) [inline widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/core/factory/SystemTabsViewFactory.kt:200]: revert this spacing please

File: widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/core/factory/SystemTabsViewFactory.kt [hunk 3]
```diff
@@ -105,6 +119,7 @@ actual class SystemTabsViewFactory actual constructor(
 
             applyPaddingIfNeeded(contentPadding)
 
+
```
Comment (Alex009) [inline widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/core/factory/SystemTabsViewFactory.kt:122]: revert this spacing please

File: widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/core/widget/TabsWidget.kt (hunk 1 above)
Comment (DevTchernov) [reply widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/core/widget/TabsWidget.kt]: Ok

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/core/factory/SystemTabsViewFactory.kt (hunk 2 above)
Comment (DevTchernov) [reply widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/core/factory/SystemTabsViewFactory.kt]: +

File: widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/core/factory/SystemTabsViewFactory.kt (hunk 3 above)
Comment (DevTchernov) [reply widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/core/factory/SystemTabsViewFactory.kt]: done

**Pull request merged**